### PR TITLE
Update mower_comms.cpp

### PR DESCRIPTION
**Pull request by midevil**
Added blade rotation change at random

### CODE REVIEW

**Comment by midevil** [comment]
thought it would be usefull to random change the mowing motor direction so both sides of the blades are used.

File: src/mower_comms/src/mower_comms.cpp [hunk 1]
```diff
@@ -107,7 +107,7 @@ void publishActuators() {
     }
 
     if(mow_xesc_interface) {
-        mow_xesc_interface->setDutyCycle(speed_mow);
+        mow_xesc_interface->setDutyCycle(speed_mow*((rand() % 2) * 2 - 1));
```
Comment (olliewalsh) [inline src/mower_comms/src/mower_comms.cpp:110]: I was about to create a PR for this, but set an initial direction at random and then just flipped it each time  setMowEnabled is called

Comment (olliewalsh) [reply src/mower_comms/src/mower_comms.cpp:110]: I think this would be flipping direction every time publishActuators is called, 20ms IIRC

**Review by olliewalsh — CHANGES_REQUESTED** [review]

File: src/mower_comms/src/mower_comms.cpp (hunk 1 above)
Comment (midevil) [reply src/mower_comms/src/mower_comms.cpp:110]: I pushed this to my mower yesterday, along with a tacho monitoring. I activated the mower by calling setMowEnabled and afaik it just worked. But yeah, if publishActuators is called every 20ms, than this is not the way to go.

**Comment by midevil** [comment]
I'll close the PR the comment of @olliewalsh is valid.

**Pull request closed without merging**